From bf3738419dce202ac63d2f5a3e34c8431008da26 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 4 Mar 2020 19:19:01 +0200
Subject: [PATCH 127/148] hide vertical divider lines in Settings

	modified:   core/res/res/values/colors_device_defaults.xml
	modified:   packages/SettingsLib/res/values/colors.xml
---
 core/res/res/values/colors_device_defaults.xml | 4 ++--
 packages/SettingsLib/res/values/colors.xml     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 7a8f411992c..0bbcac62fff 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -46,8 +46,8 @@
     <color name="error_color_device_default_dark">@color/error_color_material_dark</color>
     <color name="error_color_device_default_light">@color/error_color_material_light</color>
 
-    <color name="list_divider_color_light">#ffdadce0</color>
-    <color name="list_divider_color_dark">#85ffffff</color>
+    <color name="list_divider_color_light">#00ffffff</color>
+    <color name="list_divider_color_dark">#00ffffff</color>
     <color name="list_divider_opacity_device_default_light">@android:color/white</color>
     <color name="list_divider_opacity_device_default_dark">@android:color/white</color>
 
diff --git a/packages/SettingsLib/res/values/colors.xml b/packages/SettingsLib/res/values/colors.xml
index 5e8779fa289..b65ffe3ce1c 100644
--- a/packages/SettingsLib/res/values/colors.xml
+++ b/packages/SettingsLib/res/values/colors.xml
@@ -18,7 +18,7 @@
     <color name="disabled_text_color">#66000000</color> <!-- 38% black -->
 
     <color name="usage_graph_dots">@*android:color/tertiary_device_default_settings</color>
-    <color name="list_divider_color">#64000000</color>
+    <color name="list_divider_color">#00ffffff</color>
 
     <color name="bt_color_icon_1">#b4a50e0e</color> <!-- 72% Material Red 900 -->
     <color name="bt_color_icon_2">#b40d652d</color> <!-- 72% Material Green 900 -->
-- 
2.17.1

